OpenLIFUPrePlanning/OpenLIFUPrePlanning.py

```diff
@@ -326,20 +327,16 @@ def onLockModified(self, caller, event):
         self.updateEditTargetEnabled()
 
     def clearVirtualFitResultsIfAny(self,target: vtkMRMLMarkupsFiducialNode, reason:str):
-        """Clear virtual fit results for the target from the scene if any, and show a message dialog for the user to confirm deletion.
+        """Clear virtual fit results for the target from the scene if any.
         """
         target_id = fiducial_to_openlifu_point_id(target)
         session = get_openlifu_data_parameter_node().loaded_session
         session_id = None if session is None else session.get_session_id()
         
         if list(get_virtual_fit_result_nodes(target_id, session_id)):
-            if slicer.util.confirmYesNoDisplay(
```

Comment:
Notice that we now delete the virtual fit results without confirmation! I think this makes sense to do. It wouldn't make sense to keep the virtual fit results after the user invalidated them, unless we want to somehow undo the user's move -- not a path I want to go down! If we want to prevent this from happening accidentally then that needs to be done at the level of unlocking a target for movement, which is a separate matter from what this PR is handling